keyGenerartor/src/base/KeyGenerartor.java

```diff
@@ -0,0 +1,18 @@
+package base;
+
+import java.util.Random;
+
+public class KeyGenerartor {
```

Comment:
The name of the class is mistyped, should be:
```suggestion
public class KeyGenerator {
```